Jenkinsfile

```diff
@@ -1,39 +1,4 @@
 #!/usr/bin/env groovy
+@Library('jenkins-library@testing_integration') _
```

Comment:
please change the library pointing to the "master" for now, as we already merged the working `ci-deploy.sh` there.